tests/bin/test_busd.py

```
@@ -1,169 +1,145 @@
-        # 1. busdを起動
-        # 2. busctl spawnでタスク投函
-        # 3. tmux paneが作成されることを確認
-        # 4. busctl postで結果投函
-        # 5. logs/bus.jsonlに記録されることを確認
-        pass
+    with tempfile.TemporaryDirectory() as test_root:
+        # busctl実行
+        os.environ['ROOT'] = test_root
+        
+        # spawnメッセージ投函
+        os.system(f'./bin/busctl spawn --task T001 --goal "test"')
```

Ruff (0.12.2)

Starting a process with a shell, possible injection detected

(S605)